<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiContainer</class>
 <widget class="QWidget" name="GuiContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**************整体框体背景********/
QWidget#GuiContainer{
	background: black;	
}
/**********原理图目录背景**********/
QWidget#page_schematicDirectory{
	border-image: url(:/Schematic/Image/Schematic/schematicdirectory.png);
}
/********窗体上方条目标题如LCU，微机柜(一架)***********/
QLabel#lbl_LCU,#lbl_MicrocomputerCabinetOne,#lbl_MicrocomputerCabinetTwo,#lbl_mainEngine,#lbl_date{
	color:#dddddd;
	font: 75 15px;
	background:#1790fe;
	border:1px solid #c4c1c4;
}

QLabel#lbl_LCUState,#lbl_MicrocomputerCabinetOneState,#lbl_MicrocomputerCabinetTwoState,#lbl_mainEngineState{
	border:1px solid #c4c1c4;
	font:75 30px;
}
QLabel#lbl_dateYMD,#lbl_dateHMS{
	border:1px solid #c4c1c4;
	font:75 14px;
	color:#00f800;
}

/**********************窗体下方的功能按键****************/
QPushButton:focus{
	padding:-4px;
}

QPushButton#pbt_mainWidget,#pbt_LCU,#pbt_microcomputerCabinet,#pbt_GCU,#pbt_troubleRecord,#pbt_ladderDiagram,#pbt_schematicDiagram,#pbt_storage,#pbt_unknow,#pbt_return{
	font:75 16px;
	color:#fefe00;
	border:1px solid #0000fe;
	border-radius:8px;
}
/****按键获取焦点******/
QPushButton#pbt_mainWidget:focus,#pbt_LCU:focus,#pbt_microcomputerCabinet:focus,#pbt_GCU:focus,#pbt_troubleRecord:focus,#pbt_ladderDiagram:focus,#pbt_schematicDiagram:focus,#pbt_storage:focus,#pbt_unknow:focus,#pbt_return:focus{
	background:#0000fe;
	border-color:#fefe00;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_LCU">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>LCU</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_LCUState">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_MicrocomputerCabinetOne">
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>微机柜
(一架)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_MicrocomputerCabinetOneState">
       <property name="minimumSize">
        <size>
         <width>106</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>106</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_MicrocomputerCabinetTwo">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>微机柜
(二架)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_MicrocomputerCabinetTwoState">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_mainEngine">
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>44</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>主机</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_mainEngineState">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_date">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>日期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_dateYMD">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_dateHMS">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_guiContainer">
     <widget class="QWidget" name="page_mainWidget">
      <layout class="QGridLayout" name="gridLayout_mainWidget">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget_mainWidget">
         <widget class="QWidget" name="page_primaryWidget">
          <layout class="QGridLayout" name="gridLayout_primaryWidget">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_secondaryWidget">
          <layout class="QGridLayout" name="gridLayout_secondaryWidget">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_failureList">
          <layout class="QGridLayout" name="gridLayout_failureList">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_failureDetails">
          <layout class="QGridLayout" name="gridLayout_failureDetails">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_LCU">
      <layout class="QGridLayout" name="gridLayout_LCU">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget_LCU">
         <widget class="QWidget" name="page_busIdWidget">
          <layout class="QGridLayout" name="gridLayout_busIdWidget">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_interfaceLampHole">
          <layout class="QGridLayout" name="gridLayout_interfaceLampHole">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_DCU">
      <layout class="QGridLayout" name="gridLayout_DCU">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_GCU">
      <layout class="QGridLayout" name="gridLayout_GCU">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget_GCUAndLG">
         <widget class="QWidget" name="page_gcuWidget">
          <layout class="QGridLayout" name="gridLayout_gcuWidget">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_lgWidget">
          <layout class="QGridLayout" name="gridLayout_lgWidget">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_failureRecord">
      <layout class="QGridLayout" name="gridLayout_failureRecord">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_ladderDiagram">
      <layout class="QGridLayout" name="gridLayout_ladderDiagram">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page_schematic">
      <layout class="QGridLayout" name="gridLayout_schematic">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget_schematic">
         <widget class="QWidget" name="page_schematicDirectory">
          <layout class="QGridLayout" name="gridLayout_schematicDirectory">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_schematicDisplay">
          <layout class="QGridLayout" name="gridLayout_schematicDisplay">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_importAndExport">
      <layout class="QGridLayout" name="gridLayout_importAndExport">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_mainWidget">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>主页面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_LCU">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>LCU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_microcomputerCabinet">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>微机柜</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_GCU">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>GCU/列供</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_troubleRecord">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>故障记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_ladderDiagram">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>梯形图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_schematicDiagram">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>原理图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_storage">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>转储</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_unknow">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_return">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>77</width>
         <height>38</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
